src/components/Test.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from 'react';
+import { fb } from '../lib/firebase';
+
+const Test = () => {
+  const db = fb.firestore();
+
+  const [test, setTest] = useState([]);
+
+  const handleClick = (event) => {
+    event.preventDefault();
+    db.collection('tests').add({
+      testMessage: 'Hello World!',
+    });
+  };
+
+  useEffect(() => {
+    const unsubscribe = db.collection('tests').onSnapshot((snapshot) => {
+      const allTests = snapshot.docs.map((doc) => ({
+        id: doc.id,
+        ...doc.data(),
+      }));
+      setTest(allTests);
+    });
+    return () => {
+      console.log('cleanup');
```

Comment:
Can remove, I think. It's best not to leave `console.log` in, unless it's for a very good reason.